test/datasets/test_dataset_metadata.py

```diff
@@ -551,6 +555,40 @@ def test_output_metadata_details_when_long_set_to_true(
         dataset_metadata.output_metadata_details(long=True)
 
         # ASSERT
+        # SETUP
```

Comment:
Looks like you have two setup/assert section, I think the bits above here want removing?

Reply:
Yep, must have copied over it without realising 😄